test/decorator-annotator_test.ts

```diff
@@ -442,6 +442,21 @@ class ViewUtils {
 `);
     });
 
+    // Regression #674
+    it('should leave annotations not down-leveled', () => {
+      expectTranslated(`
+        /** @Annotation */ var RemoveMe: Function = undefined as any;
+
+        var KeepMe: Function = undefined as any;
+
+        @KeepMe()
+        @RemoveMe()
+        class ViewUtils {
+          constructor() {}
+        }
+        `).to.contain('@KeepMe');
```

Comment:
Please follow the pattern in the rest of the file and compare to the actual output.